prow/config.yaml

```diff
@@ -2085,3 +2085,57 @@ periodics:
       env:
       - name: TEST_RUN_ALL_TESTS
         value: "true"
+- cron: "6 4 * * *"
```

Comment:
@JeromeJu  curious why the cron time is so specific

Reply:
I intended to follow what's done by [periodic-tekton-catalog-integration-tests](https://github.com/tektoncd/plumbing/commit/ce9c2967903ea2dcd070a5c3cf826b04f44658eb) as it sets off the difference from [02:05 am](https://github.com/tektoncd/plumbing/blob/0cf9834f1672a1a14d4f377de86124df369040a8/prow/config.yaml#L1984) with 1 hr and 1 minute; which I interpret as it wants to have 60 minutes in between each cron job?

Reply:
ok, interesting